wrappers/python/nntile/layer/llama_attention.py

```diff
@@ -1517,3 +1517,169 @@ def to_torch_with_grads(self) -> LlamaAttention_torch:
                 to_numpy(self.in_proj_bias_v.grad).T.flatten()
             )
         return torch_layer
+
+    def get_layer_forward_flops(self):
+        total_forward_flops = 0
+        # Compute Q_transposed
+        # gemm_async(
```

Comment:
Instead of putting into comment this gemm it would be much more informative to put comments above gemm:
```
# Q_transposed = einsum('ijkl,lmn->ijkmn', W_Q, X_Q)
# gemm (kv_group_size, n_head_kv, head_size, n_emb)
# by (n_emb, n_seq, n_batch)
# into (kv_group_size, n_head_kv, head_size, n_seq, n_batch)
```
One can easily guess all the shapes and how to get FLOP count without a mistake